pkg/deployments/tasks/20220318-batch-relayer-v2/test/task.fork.ts

```diff
@@ -0,0 +1,159 @@
+import hre from 'hardhat';
+import { expect } from 'chai';
+import { BigNumber, Contract } from 'ethers';
+
+import { BigNumberish, fp } from '@balancer-labs/v2-helpers/src/numbers';
+
+import Task from '../../../src/task';
+import { getForkedNetwork } from '../../../src/test';
+import { getSigner, impersonate, impersonateWhale, setBalance } from '../../../src/signers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { RelayerAuthorization, SwapKind, WeightedPoolEncoder } from '@balancer-labs/balancer-js';
+import { fromNow, MINUTE } from '@balancer-labs/v2-helpers/src/time';
+import { MAX_UINT256 } from '@balancer-labs/v2-helpers/src/constants';
+
+describe('BatchRelayerLibrary', function () {
+  const task = Task.forTest('20220318-batch-relayer-v2', getForkedNetwork(hre));
+
+  let relayer: Contract, library: Contract;
+  let sender: SignerWithAddress, admin: SignerWithAddress;
+  let vault: Contract, authorizer: Contract, dai: Contract, usdc: Contract;
+
+  const DAI = '0x6b175474e89094c44da98b954eedeac495271d0f';
+  const USDC = '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48';
+
+  const DAI_USDC_USDT_POOL = '0x06df3b2bbb68adc8b0e302443692037ed9f91b42000000000000000000000063';
+  const ETH_DAI_POOL = '0x0b09dea16768f0799065c475be02919503cb2a3500020000000000000000001a';
+
```

Comment:
Are these just the regular batch relayer tests? It'd be good to test the new functionality here instead of this.